docs/nx-cloud/reference/release-notes.md

```diff
@@ -1,5 +1,69 @@
 # Enterprise Release Notes
 
+### 2024.10
+
+This is a big release so let's go through the highlights first. There is also an important "Breaking changes" section at the end.
+
+## New version structure
+
+We have changed our version structure to a more simplified tag: `YEAR.MM.PATCH_NUMBER`
+The goal is to make it easier to spot how old/recent your existing NxCloud version is and compare it against newer deployments.
+
+## DTE summary table on main agent
+
+When distributing with DTE, up until now, we have been replaying all your tasks logs "as they come in" from the DTE agents back onto your main job.
+
+This is not that useful on big workspaces, with large task affected task graphs as it can be hard to follow the all the outputs from all the agents streaming back concurrently.
```

Comment:
```suggestion
This is not that useful on big workspaces, with large task affected task graphs as it can be hard to follow all the outputs from all the agents streaming back concurrently.
```